core/jobs/batch_jobs/exp_version_history_computation_job.py

```diff
@@ -126,8 +126,10 @@ def generate_exploration_from_snapshot(
                 snapshot_dict = snapshot_model.content
                 exp_id = snapshot_model.get_unversioned_instance_id()
                 model_class = exp_models.ExplorationModel
-                reconstituted_model = model_class(id=exp_id)._reconstitute(  # pylint: disable=protected-access
-                    snapshot_dict
+                reconstituted_model = (
+                model_class( # pylint: disable=protected-access
```

Comment:
Please fix the indentation. Any time a line above ends with '(', the following line must be indented 4 spaces forward. Any time a line starts with ')' it should similarly be pushed 4 spaces backward from the preceding line.

Ditto below.